src/components/v2/Notice/index.tsx

```diff
@@ -0,0 +1,48 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import { Icon } from 'components';
+
+import { NoticeVariant, useStyles } from './styles';
+
+interface INoticeProps {
+  className?: string;
+  title?: string;
+  description: string;
+  variant?: NoticeVariant;
+}
+
+const getNoticeIconName = (variant: NoticeVariant): 'info' | 'notice' | 'checkInline' => {
+  switch (variant) {
+    case 'error':
+      return 'notice';
+    case 'success':
+      return 'checkInline';
+    default:
+    case 'info':
+      return 'info';
+  }
+};
+
+export const Notice = ({ className, title, description, variant = 'info' }: INoticeProps) => {
```

Comment:
We export our variants similar to how we export variants for buttons

Reply:
added `types.ts`